doc/nrf/includes/tfm_spm_thingy91.txt

```diff
@@ -1,8 +1,12 @@
 The sample is configured to compile and run as a non-secure application.
-Therefore, it automatically includes an additional secure firmware component, that prepares the required peripherals and secure services to be available for the application.
+Therefore, it automatically includes an additional secure firmware component that prepares the required peripherals and secure services to be available for the application.
 
-When built for the ``nrf9160dk_nrf9160_ns`` build target, the sample automatically includes :ref:`Trusted Firmware-M <ug_tfm>`.
+For nRF9160 DK, when the sample is built using the ``nrf9160dk_nrf9160_ns`` build target, it automatically includes :ref:`Trusted Firmware-M <ug_tfm>`.
 You can also configure it to use the :ref:`secure_partition_manager` instead of Trusted Firmware-M.
 
-When built for the ``thingy91_nrf9160_ns`` build target, the sample automatically includes the :ref:`secure_partition_manager`.
+For Thingy:91, when the sample is built using the ``thingy91_nrf9160_ns`` build target, it automatically includes the :ref:`secure_partition_manager`.
 You can also configure it to use :ref:`TF-M <ug_tfm>` instead of Secure Partition Manager.
+
+.. note::
+   TF-M is not completely supported for Thingy:91 v1.5.0 and lower versions because it does not support when upgrading the firmware using MCUboot.
```

Comment:
It could perhaps be reworded to something like this:
```
TF-M is not supported for Thingy:91 v1.5.0 and lower versions when using the factory-programmed bootloader to upgrade the firmware.
TF-M works for all Thingy:91 versions if you first upgrade the bootloader using an external debug probe. 
After upgrading the bootloader to NCS v2.0.0, TF-M will work also when using the bootloader to upgrade the firmware.
```

This note can be removed when https://github.com/nrfconnect/sdk-trusted-firmware-m/pull/51 is merged.

Reply:
@jtguggedal : As this a known issue for 2.0 release, I have added this to the known issue list and provided a reference from this note to the known issue. Please review.

I would make sure that the note is removed once https://github.com/nrfconnect/sdk-trusted-firmware-m/pull/51 is merged.

Reply:
Alright, sounds good!